### ISSUE TITLE
[Bug][a11] Save to collection button touch target is too small

### ISSUE THREAD

**Issue by sv-ohorvath** [mozilla-mobile/fenix#8629]
## Steps to reproduce
1. Open a tab
2. Scan the homescreen with the a11y scanner.
### Expected behavior
a11y scanner suggests to increase the "Save to collection" button touch target to 48dp or larger.
### Actual behavior
Save to collection touch target is 36dp.

### Device information

* Android device: Galaxy Tab 3 (Android 8)
* Fenix version: Beta 4.0-build1

**Comment by ekager** [comment]
We could do something like changing the minheight to 48dp and adding 6dp of top and bottom inset to achieve a larger touch target with the same look for the button.

**Comment by brampitoyo** [comment]
@ekager @sv-ohorvath I think the button size used here matches the sizes of buttons used elsewhere in the app:

- Bookmarks → Sign in to see synchronised bookmarks
- History → Delete history
- Settings → Delete browsing data

I totally agree with the fact that 48dp fits Material’s recommendation. At the same time, following the spec seems like a good idea.

Perhaps following the spec should take priority here. What do you think?

**Comment by asadotzler** [comment]
IMO, we should be striving for usability for all users and that includes people with disabilities that prevent them from hitting smaller touch targets easily. All of our buttons should meet or exceed size recommendations.

**Comment by asadotzler** [comment]
If we've got a spec or design system that's telling us to make things too small, we should fix that too. Our designs and implementations should meet or exceed accessibility standards.

**Comment by mcarare** [comment]
This should be fixed. Can QA please confirm? TY!

**Comment by sflorean** [comment]
I was able to reproduce this issue on an affected build with Google Pixel 3 (Android 10) and I can confirm that no suggestions are made on the same device, build Nightly 6/3.
